txscript/sign_test.go

```diff
@@ -1440,7 +1440,7 @@ var sigScriptTests = []tstSigScript{
 		name: "one input uncompressed",
 		inputs: []tstInput{
 			{
-				txout:              wire.NewTxOut(coinbaseVal, uncompressedPkScript),
+				txout:              wire.NewTxOut(coinbaseVal, uncompressedPkScript, wire.TokenData{}),
```

Comment:
Signing tests should definitely include outputs with sample token data.

Reply:
Same as https://github.com/gcash/bchd/pull/545#discussion_r1852684161. Resolving it for now.